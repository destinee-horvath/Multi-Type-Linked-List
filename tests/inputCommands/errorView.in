VIEW 
VIEW
VIEW -10
view 0
ViEW 0
VIEW abc