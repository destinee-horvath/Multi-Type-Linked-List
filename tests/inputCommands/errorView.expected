INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: VIEW
